rule/templates/_rule.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
     },
     fixable: null, // Or `code` or `whitespace`
     schema: [], // Add a schema if the rule has options
+    messages: {} // Add messageId and message
```

Comment:
Looks like there should be a trailing comma based on the prettier default:

```suggestion
    messages: {}, // Add messageId and message
```

Reply:
The comma makes sense, although I couldn't see any prettier in this repo.

Reply:
Yeah, doesn't come with that, but the goal is for our formatting to be already compatible with it.